src/backend/src/services/teams.services.ts

```diff
@@ -162,4 +162,59 @@ export default class TeamsService {
 
     return teamTransformer(updateTeam);
   }
+
+  /**
+   * Creates a new team in the database
+   * @param submitter The submitter who is trying to create a new team
+   * @param teamName the name of the new team
+   * @param headId the id of the user who will be the head on the new team
+   * @param slackId the slack id for the slack channel for the team
+   * @param description a short description of the team (must be less than 300 words)
+   * @returns The newly created team
+   */
+  static async createTeam(
+    submitter: User,
+    teamName: string,
+    headId: number,
+    slackId: string,
+    description: string
+  ): Promise<Team> {
+    if (!isAdmin(submitter.role)) {
+      throw new AccessDeniedException('You must be an admin or higher to create a new team!');
+    }
+
+    if (!isUnderWordCount(description, 300)) throw new HttpException(400, 'Description must be less than 300 words');
```

Comment:
I feel like we shouldmake this a function because we use it elsewhere but maybe we can make a backend refactoring epic and add stuff like this to it

Reply:
I can do it here, but we should make the backend refactoring epic